Source/GUI/MemWatcher/MemWatchWidget.h

```diff
@@ -39,6 +39,7 @@ class MemWatchWidget : public QWidget
   void copySelectedWatchesToClipBoard();
   void cutSelectedWatchesToClipBoard();
   void pasteWatchFromClipBoard(const QModelIndex& referenceIndex);
+  void copyAddressToClipboard(u32 addr);
```

Comment:
Technically, `copyAddressToClipboard()` could be `static`: it doesn't reference `this`. And, arguably, the function is general enough to be placed in a common module that could be reused later (although I don't have any suggestion).

Reply:
This method is no longer present, see [above](https://github.com/aldelaro5/dolphin-memory-engine/pull/167#discussion_r1667768094)